src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -20,23 +20,22 @@ public class Cart {
     @Column(name = "cartId", unique = true)
     private Long id;
 
-    @OneToOne(mappedBy = "cart")
+    @OneToOne(cascade = CascadeType.ALL, mappedBy = "cart")
```

Comment:
To be consulted with other pull-request (JDP230502-19) - if merged. If not, I suggest to check if other parameters of cascade would be more suitable.

Reply:
JDP230502-19 is merged. Here cascadeType is the same there.